prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -607,7 +607,7 @@ ssize_t efa_rdm_ep_post_handshake(struct efa_rdm_ep *ep, struct efa_rdm_peer *pe
 
 	pkt_entry = efa_rdm_pke_alloc(ep, ep->efa_tx_pkt_pool, EFA_RDM_PKE_FROM_EFA_TX_POOL);
 	if (OFI_UNLIKELY(!pkt_entry)) {
-		EFA_WARN(FI_LOG_EP_CTRL, "PKE entries exhausted.\n");
+		EFA_DBG(FI_LOG_EP_CTRL, "PKE entries exhausted for posting handshake.\n");
```

Comment:
Hmm, this kind of implies there's a separate PKE pool for handshakes. Not a huge deal.

Reply:
not really, I can remove the extra wording